Kube/scripts/install-shield.sh

```diff
@@ -202,13 +225,11 @@ function generate_rancher_token() {
 
 # Wait until Rancher is launched
 function wait_for_rancher() {
-    #read es_rancher files
     #read es_rancher files
     if [ -f $RANCHER_URL_FILE ]; then
         RANCHER_SERVER_URL=$(cat $RANCHER_URL_FILE)
     else
-        MY_IP="$(ip route get 8.8.8.8 | awk -F"src " 'NR==1{split($2,a," ");print a[1]}')"
-        RANCHER_SERVER_URL="https://$MY_IP:8443"
+        RANCHER_SERVER_URL="https://$(get_my_ip):8443"
```

Comment:
@BenyH are you sure ?

Reply:
yes this is @briskycat fix for Rancher with US Proxy

Reply:
We had an error here with an upstream proxy since finding a route to 8.8.8.8 is not always possible in such environments.